### DISCUSSION TITLE
My account restricted, what can I do with it?

### DISCUSSION THREAD

**Discussion by khoahnd** [codewars/codewars.com#3146] [Q&A] [answered]
Hello Codewars Admins and Community,

I'm contacting you with an urgent concern regarding my [Codewars account](https://www.codewars.com/users/khoahnd), which has unexpectedly been restricted. I need clarification on this situation as I am unaware of any breach of the terms and conditions of the platform.

To my knowledge, I've always adhered to community guidelines and ethical coding practices. Therefore, this restriction is unexpected and needs to be clarified.

I kindly request the Codewars team or any community member who can assist to provide the following:

1. Specific details on why my account was restricted.
2. Guidance on resolving this issue and restoring my account.
3. Tips to avoid potential misunderstandings in the future.

I am eager to resolve this quickly and continue my journey with Codewars. Any help or clarification on this would be greatly appreciated.

Thank you for your time and assistance.

**Comment by Kacarott** [comment]
I was not the one who restricted your account, but I can answer some questions:
1. Your account was restricted because many of your solutions are copies of others solutions online. Copying solutions is cheating.
2. The easiest way to restore would be to make a new account and start fresh, but you can also wait for the mod who restricted you to see their decision.
3. If you don't want your account restricted in the future, then don't cheat and plagiarise others solutions as your own :)

**Reply by khoahnd** [reply]
I have referenced solutions from others to compare with my solution, but I did not copy them verbatim, and I also want to know where I cheated.

**Reply by Kacarott** [reply]
Comparing another's solution to your own has nothing to do with submitting someone else's solution as your own. If you want to compare to other's solutions, then submit your own solution first, then in the solutions page you can freely see and compare everyone elses.

Anyway, as said, the mod who restricted you will see this post sometime soonish and can re-evaluate.

**Reply by Blind4Basics** [reply, marked as answer]
Hello,

There is nothing to reevaluate. Delete the account start fresh, and this time don't use solutions you find on the internet. Note that rewritting solutions you find on the internet will lead to the very same situation. Write your own logic, otherwise this will just repeat again.

Cheers
